tests/sentry/api/endpoints/test_organization_releases.py

```diff
@@ -911,6 +914,132 @@ def test_bad_repo_name(self):
         assert response.data == {'refs': [u'Invalid repository names: not_a_repo']}
 
 
+class OrganizationReleaseCommitRangesTest(SetRefsTestCase):
+    def setUp(self):
+        super(OrganizationReleaseCommitRangesTest, self).setUp()
+        self.url = reverse('sentry-api-0-organization-releases',
+                           kwargs={'organization_slug': self.org.slug})
+
+    @patch('sentry.tasks.commits.fetch_commits')
+    def test_simple(self, mock_fetch_commits):
+        refs = [
+            {
+                'repository': 'test/repo',
+                'previousCommit': None,
+                'commit': 'previous-commit-id..current-commit-id',
+            },
+            {
+                'repository': 'test/repo',
+                'previousCommit': 'previous-commit-will-be-ignored',
+                'commit': 'previous-commit-id-2..current-commit-id-2',
+            },
+            {
+                'repository': 'test/repo',
```

Comment:
Should we allow multiple commit ranges from the same repository? That seems like a situation that is a logical error that the user has made.